### ISSUE TITLE
2 features one is a bit of an issue for samsung

### ISSUE THREAD

**Issue by ravensousa** [futo-org/grayjay-android#586]
1st feature zoomable videos ie like YouTube 2nd feature/issue in Samsung phones the smart pop up resets the app so if I go from small floating window into bubble or full screen the app fully refreshes kicking me off my video this is not pip this is the smart pop up Samsung feature under Samsung notifications advanced tab I'd really like better support for it as sometimes I don't use pip or I get a notification open it and later want to full-screen it

**Comment by Zvonimir-FUTO** [comment]
Closing issue as stale.

**Comment by ravensousa** [comment]
> @ravensousa you can now zoom in fullscreen mode. Regarding the Samsung's smart pop up feature, are you still experiencing issues with that on latest version of Grayjay?

Yes I'm still having the issue with amrt pop up sorry I'm replying so late I never actually got the email you replied

**Comment by defiling9046** [comment]
Close as stale? @Zvonimir-FUTO
